docs/source/virtual_rainforest/core/axes.md

```diff
@@ -0,0 +1,62 @@
+---
+jupytext:
+  formats: md:myst
+  main_language: python
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: vr_python3
+  language: python
+  name: vr_python3
+---
+
+# Core axes
+
+The Virtual Rainforest uses a set of **core axes** which have dimensions and possibly
+coordinates set by the configuration of a simulation. Each core axis is defined below
+along with the different validation routines used to map data onto a core axis.
+
+Data is loaded into the simulation as {class}`~xarray.DataArray` object. Each of the
+dimensions of a data array will have a _dimension name_ and may also have _coordinate
+values_ assigned for the dimension. In general, validation for a given core axis will
+look for specific dimension names and possibly coordinate values on those dimensions and
+then attempts to map those dimensions onto the configured axis.  If an input data set
+uses **any** of the dimension names associated with a given core axes, then that data
+must past validation on that axis.
+
+## The `spatial` core axis
+
+```{admonition} Array dimensions
+This core axis looks for: `cell_id`, `x` and `y`. We expect Virtual Rainforest to be 
+used with projected coordinate systems and so `lat` and `long` dimensions do not cause 
+data to be mapped onto the `spatial` axis.
```

Comment:
This sentence is not clear to me

Reply:
I've clarified this - we just don't try and validate `lat` and `long` dimensions onto the `spatial` axis. Actually, we could add an 'AxisValidator' that explicitly looks for those names and then always fails validation on the spatial axis. It's a bit of an odd use case though! 

@dalonsoa - what do you think? Having something that explicitly bails when it encounters latitude and longitude dimensions with a meaningful error seems preferable to just having the data silently not get mapped onto an axis that it clearly is expected to match? It's an odd use for an `AxisValidator` - to always fail when it encounters `lat`, `long` etc, but it _is_ meaningfully axis validation!